File: Toast/Toast/src/main/java/se/kau/android/toast/MainActivity.java
package se.kau.android.toast;

import androidx.appcompat.app.AppCompatActivity;

import android.util.Log;

import android.os.Bundle;
import android.view.View;
import android.widget.*;

public class MainActivity extends AppCompatActivity {

    final static String TAG ="click";
    Button clickButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        clickButton = findViewById(R.id.myButton);
    }

    public void clickMe(View v){
        Log.d(TAG, "Its clicking .. HELLO WORLD!");
        Toast.makeText(MainActivity.this, "HELLO WORLD!", Toast.LENGTH_SHORT).show();
    }
}

File: BlocketArab/app/src/main/java/MainActivity.kt
package com.example.myapp

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.example.blocketarab.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)  // تعيين الواجهة للنشاط

        // العثور على الأزرار وتعيين مستمعات لها
        val viewToolsButton = findViewById<Button>(R.id.viewToolsButton)
        val logoutButton = findViewById<Button>(R.id.logoutButton)

        // تعيين مستمعين للأزرار
        viewToolsButton.setOnClickListener {
            // هنا يمكنك إضافة كود لعرض قائمة الأدوات
        }

        logoutButton.setOnClickListener {
            // هنا يمكنك إضافة كود لتسجيل الخروج
        }
    }
}
